src/App.types.ts

```diff
@@ -0,0 +1,36 @@
+export type HackerState = {
+    id: string;
+    email: string;
+    firstName: string;
+    lastName: string;
+    age: number;
+    gender: string;
+    ethnicity: string;
+    shirtSize: string;
+    schoolName: string;
+    major: string;
+    year: string;
+    schoolStanding: string;
+    dietaryRestrictions: string;
+    allergies: string;
+    numberOfPreviousHackathons: number;
+    resume: string;
+    linkedIn: string;
+    gitHub: string;
+    devpost: string;
+    otherSites: string;
+    teamCode: string;
+    teamName: string;
+    teammates: [string];
```

Comment:
`string[]` is array of strings, [string] means an array with a single element, which is string

Reply:
this is from old code that is not gonna be used in production